x-pack/plugins/infra/public/alerting/metric_threshold/mocks/metric_threshold_rule.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Rule } from '@kbn/alerting-plugin/common';
+import { v4 as uuidv4 } from 'uuid';
+import { Aggregators, Comparator } from '../../../../common/alerting/metrics';
+import { MetricThresholdRuleTypeParams } from '..';
+
+export const buildMetricThresholdRule = (
+  rule: Partial<Rule<MetricThresholdRuleTypeParams>> = {}
+): Rule<MetricThresholdRuleTypeParams> => {
+  return {
+    alertTypeId: 'metrics.alert.threshold',
+    createdBy: 'admin',
+    updatedBy: 'admin',
+    createdAt: new Date('2023-02-20T15:25:32.125Z'),
+    updatedAt: new Date('2023-03-02T16:24:41.177Z'),
+    apiKey: 'apiKey',
```

Comment:
When I logged this information in the browser, this field didn't exist. Not sure if we need to update the type or not.

@elastic/response-ops What do you think?